modules/apps/29-fee/keeper/msg_server_test.go

```diff
@@ -0,0 +1,32 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+)
+
+func (suite *KeeperTestSuite) TestRegisterCounterpartyAddress() {
+	validAddr := suite.chainA.SenderAccount.GetAddress().String()
+	validAddr2 := suite.chainB.SenderAccount.GetAddress().String()
+
+	testCases := []struct {
+		msg     *types.MsgRegisterCounterpartyAddress
+		expPass bool
+	}{
+		{
+			types.NewMsgRegisterCounterpartyAddress(validAddr, validAddr2),
+			true,
+		},
+	}
+
+	for _, tc := range testCases {
+		suite.SetupTest()
+		_, err := suite.chainA.SendMsgs(tc.msg)
```

Comment:
@colin-axner pushing this up to get some feedback. I am trying to write a test here to test message execution and I'm receiving an error here:

```
	fee payer address: cosmos16qqs8alawtkthp0ljmq3ggx8kaud7paaazdwh4 does not exist: unknown address
```

It's failing on the `getSigners` step. I'm assuming it can't find the address defined on line 8. 

Any ideas?

Reply:
try using the `suite.chainA.SenderAccount.GetAddress().String()` and `suite.chainB.SenderAccount.GetAddress().String()` as your valid addresses

Reply:
The test is failing because the sender account needs to exist within the auth account keeper. It needs an associated signing sequence/balance for the tx to be processed

Reply:
https://github.com/cosmos/ibc-go/pull/391/files#diff-d7b50e4eedd25ea511cce1896451c609c0ccdcb5e93f1c96c59b05d6567e2cbbR8

I believe that is what I am doing here?

Reply:
> The test is failing because the sender account needs to exist within the auth account keeper. It needs an associated signing sequence/balance for the tx to be processed

Ah interesting!

Reply:
Updated, working now :+1: